src/averell/utils.py

```diff
@@ -15,6 +15,7 @@
 DEFAULT_OUTPUT_FOLDER = Path.cwd() / "corpora"
 
 TEI_NAMESPACE = "{http://www.tei-c.org/ns/1.0}"
+XML_NS = "{http://www.w3.org/XML/1998/namespace}"
```

Comment:
So much namespacing, I guess that's needed for XPath.

Reply:
Yes, we need this to find the elements that have an `xml:id `attribute